src/sas/sasgui/perspectives/fitting/basepage.py

```diff
@@ -2667,17 +2667,18 @@ def _lay_out(self):
 
     def _find_polyfunc_selection(self, disp_func=None):
         """
-        FInd Comboox selection from disp_func
+        Find Combobox selection from disp_func
 
         :param disp_function: dispersion distr. function
         """
         # Find the selection
         if disp_func is not None:
             try:
-                return POLYDISPERSITY_MODELS.values().index(disp_func.__class__)
+                index = POLYDISPERSITY_MODELS.values().index(disp_func.__class__)
+                return index, POLYDISPERSITY_MODELS.keys()[index]
             except ValueError:
```

Comment:
Can you use ```return index, disp_func.type``` instead?

Note:```dict.values().index()``` does not work in python 3.  You could instead use the following, which will work for both:
```
index = list(POLYDISPERSITY_MODELS).index(disp_func.type)
```
This works because ```list(OrderedDict)``` is just the list of the keys, which in this case are created from ```distribution.type``` for each of the distributions in the ordered dict.